dex/src/instruction.rs

```diff
@@ -429,11 +446,25 @@ pub enum MarketInstruction {
     /// 3. `[writable]` OpenOrders
     /// 4. `[signer]` the OpenOrders owner
     /// 5. `[writable]` event_q
+    CancelOrderNoErrorV2(CancelOrderNoErrorInstructionV2),
```

Comment:
Nit: But should this be CancelOrderV2NoError?

Reply:
yeah that makes more sense. ill replace all NoErrorV2 with V2NoError

only downside is that if we upgrade a NoError instruction without upgrading the normal one it'll be CancelOrderV2NoErrorV2 😝 very low probability that we'll do that as opposed to upgrading the normal cancel instructions tho